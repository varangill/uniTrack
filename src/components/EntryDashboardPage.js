import React from 'react';
import EntryList from './EntryList';
import EntryListFilters from './EntryListFilters';

const EntryDashboardPage = () => (
  <div>
    <EntryListFilters />
    <EntryList />
  </div>
);

export default EntryDashboardPage;
