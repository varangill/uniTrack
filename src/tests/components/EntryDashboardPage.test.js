import React from 'react';
import { shallow } from 'enzyme';
import EntryDashboardPage from '../../components/EntryDashboardPage';

test('should render EntryDashboardPage correctly', () => {
  const wrapper = shallow(<EntryDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
