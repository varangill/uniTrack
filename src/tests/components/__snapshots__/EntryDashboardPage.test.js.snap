// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render EntryDashboardPage correctly 1`] = `
<div>
  <Connect(EntryListFilters) />
  <Connect(EntryList) />
</div>
`;
